content/docs/connect/connection-pooling.md

```diff
@@ -63,7 +63,9 @@ Even with the largest compute size, the `max_connections` limit may not be suffi
 Some applications open numerous connections, with most eventually becoming inactive. This behavior can often be attributed to database driver limitations, running many instances of an application, or applications with serverless functions. With regular Postgres, new connections are rejected when reaching the `max_connections` limit. To overcome this limitation, Neon supports connection pooling using [PgBouncer](https://www.pgbouncer.org/), which allows Neon to support up to 10,000 concurrent connections.
 
 <Admonition type="important">
-Connection pooling supports up to 10,000 concurrent connections when multiple Postgres users are connecting to multiple Postgres databases. For a single user connecting to a single database, the maximum number of concurrent connections is 64. This limit is controlled by the PgBouncer `default_pool_size` parameter, which defines the default number of server connections per user/database pair. See [Neon PgBouncer configuration settings](#neon-pgbouncer-configuration-settings). If your application connects to your database using a single Postgres role, expect to encounter this limit when exceeding the 64 concurrent connection limit. For information about creating additional Postgres roles, see [Manage roles](/docs/manage/roles#manage-roles-with-sql). Roles can created via the Neon Console, CLI, API, and SQL.
+Connection pooling supports up to 10,000 concurrent connections. However, these connections are not a magic bullet: These connections are pooled and take shared ownership of a pool of connections to PostgreSQL. This pool is shared between all connections that connect with the same user credentials to the same database, and share a total of up to [64 connections](#neon-pgbouncer-configuration-settings). This means any user can only have 64 concurrently active transactions in a single database when connected through the pooler endpoint.  
```

Comment:
> [64 connections](#neon-pgbouncer-configuration-settings). This means any user can only have 64 concurrently active transactions in a single database when connected through the pooler endpoint.

in my limited knowledge, a transaction can have any X number of operations in it to be executed in sequence, while a connection may or may not have a sequence of operations. would it make sense to update the wording to stick to either one?

basically, total of up to 64 connections or transactions?

Reply:
64 transactions. Our pgbouncer runs in transaction mode.
https://neon.tech/docs/connect/connection-pooling#neon-pgbouncer-configuration-settings

Reply:
How do we communicate it simply to the users what'd 64 transactions mean?  It's common to get lost into the details.

The current/PR'd wording fails to communicate that it's about internals of Neon itself: PgBouncer - Postgres (within Neon) which user does not have access to.

Think of it as a user who comes in with the expectation of 10K connections, pulls up this guide and then sees "For a single user connecting to a single database, the maximum number of concurrent connections is 64." which in fact is not correct as this is between PgBouncer and Postgres.

Also, while it seems great that we can open upto 10K connections, can we also document the concurrent limit of starting those connections?